js/media-modal.js

```diff
@@ -0,0 +1,65 @@
+( function ( $ ) {
+	'use strict';
+	
+	if (wp.media) {
+		wp.media.view.Modal.prototype.on('ready', function(data) {
+			wp.media.view.Modal.prototype.on( "open", function() {
+				tinymce_init();
+			});
+			wp.media.view.Modal.prototype.on( "close", function() {
+				tinymce_init();
+			});
+			let lastUrl = location.href; 
+			new MutationObserver(() => {
+				const url = location.href;
+				if (url !== lastUrl) {
+					// console.log('URL changed!', location.href);
+					lastUrl = url;
+					tinymce_init();
+				}
+				}).observe(document, {subtree: true, childList: true});
+		});
+		
+	}
+
+	function tinymce_init(){
+		tinymce.remove();
+		tinymce.init(
+			{
+				selector: "textarea.rwmb-wysiwyg",
```

Comment:
E check xem đoạn code này chạy ở những chỗ nào. Hiện tại a thấy e hook vào `wp_enqueue_media`, mà cái này có thể chạy ở nhiều chỗ khác nữa, selector lại bao gồm tất cả các textarea của mình nên sẽ gây ảnh hưởng. Tốt nhất e nên check xem mình enqueue ở đúng màn hình media kia thôi thì sẽ an toàn hơn.

Reply:
em có add_action( 'wp_enqueue_media', array( $this, 'enqueue_media_js' ), 11, 0 ); file js này ở media-modal.php nên chắc nó ko bị liên quan đến chỗ khác.